strings/ctype-bin.c

```diff
@@ -291,9 +291,15 @@ int my_wc_mb_bin(CHARSET_INFO *cs __attribute__((unused)),
 void my_hash_sort_bin(CHARSET_INFO *cs __attribute__((unused)),
                       const uchar *key, size_t len,ulong *nr1, ulong *nr2)
 {
-  const uchar *end = key + len;
```

Comment:
`/server/strings/ctype-bin.c:294:26: runtime error: applying zero offset to null pointer`